kube/services/jobs/dbgapsync-job.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: batch/v1
```

Comment:
I don't think we should create a new sync job? the existing usersync job and usersync cronjob should be changed to use the fence-create script, and use the current user yaml either from the local yaml or s3, and optionally sync from dbgap depends on a flag in configmap?